### ISSUE TITLE
Zoom while dragging

### ISSUE THREAD

**Issue by hassam7** [gmaps-marker-clusterer/gmaps-marker-clusterer#51]
If you are dragging over marker clusterer, It automatically zoom's in on cluster. It should only perform click when preceded by drag.
jsfiddle: https://plnkr.co/edit/AqXwOp44asDXjOSCcjck?p=preview
Tested on Chrome 61.0.3163.91

**Comment by Remo** [comment]
Thanks, can confirm this. I would have thought, that this line would stop the browser from doing that. Will have to debug this.

https://github.com/gmaps-marker-clusterer/gmaps-marker-clusterer/blob/master/src/markerclusterer.js#L1252

**Comment by hassam7** [comment]
A quick fix I which I found on stackoverflow is:

`this.gmaps.event.addListener(map, 'dragstart', () => {
            this.markerCluster.zoomOnClick_ = false;
          });
          this.gmaps.event.addListener(map, 'mouseup', () => {
            setTimeout(() => {
              this.markerCluster.zoomOnClick_ = true;
            }, 50);
          });`

**Comment by Remo** [comment]
Oh but that looks like it's intentional, there's even an option you can specify (`zoomOnClick`): https://plnkr.co/edit/VkaEydqffulNrwveG2QE?p=preview

**Comment by hassam7** [comment]
yes, you are right but in most cases this is unexpected behaviour I think. Whats your opinion?

**Comment by Remo** [comment]
I agree, but it would be a breaking chance and since we don't have any plans to release a new major version we should keep it that way.

**Comment by Remo** [comment]
Although, there might still be a big. The check to `zoomOnClick_` happens in `triggerClusterClick` and that method should only be called if `isDragging` is `false`, maybe still requires some debugging..
